Makefile

```diff
@@ -76,10 +76,12 @@ GOTESTS ?= ./...
 # Use "-short" to skip long tests, or "-verbose" for more verbose reporting. Run
 # go help testflags to see all options.
 GOTESTFLAGS ?= ""
-test: manifests generate vet mocks ${SIGNED_ARTIFACTS} ## Run tests.
-	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
-	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); $(GO) test $(GOTESTFLAGS) `$(GO) list $(GOTESTS) | grep -v mocks` -coverprofile cover.out
+test: manifests generate vet mocks ${SIGNED_ARTIFACTS} $(GOBIN)/setup-envtest ## Run tests.
+	source <(setup-envtest use -i -p env 1.23.x)
+	$(GO) test $(GOTESTFLAGS) `$(GO) list $(GOTESTS) | grep -v mocks` -coverprofile cover.out
+
+$(GOBIN)/setup-envtest: ## Install setup-envtest
+	go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
```

Comment:
I'd generally prefer not to use `@latest` here, so we don't get bitten by this updating unexpectedly on the build servers.

Reply:
done

Reply:
This doesn't work, I assume since it is a submodule there is some ongoing work to support this https://github.com/kubernetes-sigs/kubebuilder/issues/2480